plasma_framework/contracts/src/framework/ExitGameController.sol

```diff
@@ -120,7 +155,7 @@ contract ExitGameController is ExitGameRegistry {
             processor = delegations[uniquePriority];
         }
 
-        emit ProcessedExitsNum(processedNum, _token);
+        emit ProcessedExitsNum(processedNum, token);
```

Comment:
please add `vaultId` to this event too, or replace `token` with `exitQueueKey `

Reply:
done